opensrp-reveal/src/main/java/org/smartregister/reveal/interactor/BaseInteractor.java

```diff
@@ -130,29 +133,44 @@ public void saveJsonForm(String json) {
         eventJson.put(DETAILS, getJSONObject(jsonForm, DETAILS));
         eventClientRepository.addEvent(entityId, eventJson);
         return gson.fromJson(eventJson.toString(), org.smartregister.domain.db.Event.class);
-
-
     }
 
-    private void saveSprayForm(JSONObject jsonForm) {
+    private void saveMosquitoInterventionForm(JSONObject jsonForm) {
```

Comment:
Maybe you could name this method `saveStructureForm` so that we name using on the intervention unit.

Reply:
This does not save a structure form so I don't think the name `saveStructureForm` fits.

This saves forms that have to do with various mosquito interventions like spraying, collecting adult mosquitoes, collecting mosquito larvae or distributing nets. All these are activities that have to do with mosquitoes, hence the name.

They also happen to have a similar saving process. I think the function name is descriptive as is.